src/css/imports/display.less

```diff
@@ -44,45 +44,134 @@
     }
 }
 
-// DisplayIcon
+// ========== Display Icons ========== //
+
+.jw-display-container {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  .jw-breakpoint-0.jw-state-playing &,
+  .jw-breakpoint-0.jw-state-paused &,
+  .jw-breakpoint-0.jw-state-buffering &,
+  .jw-breakpoint-1.jw-state-playing &,
+  .jw-breakpoint-1.jw-state-paused &,
+  .jw-breakpoint-1.jw-state-buffering & {
+    padding-top: 14px;
+  }
+}
+
+@keyframes jw-display-icon-container-slide-up {
```

Comment:
Animations should be added to skins, not core player styles. This applies the animation to all skins, but makes it difficult to override or customize in customer skins.

Animations would be better organized as styles/mixins included in all skins.